test/unit/constants.test.ts

```diff
@@ -0,0 +1,156 @@
+      expect(contextOutput.config.protocol).toBe(fixtures.context.valid.networkConfig.protocol);
+    });
+
+    test('Should work when public key is passed along with network config', () => {
+      const contextOutput = context(
+        fixtures.context.valid.networkConfig,
+        fixtures.context.valid.publicKey
+      );
+      expect(contextOutput).not.toBeNull;
+      expect(contextOutput.publicKey).not.toBeNull;
+      expect(contextOutput.cAddressHex).toBe(fixtures.context.valid.cAddressHex);
+    });
+
+    test('Should work when public key, private key hex is passed along with network config', () => {
+      const contextOutput = context(
+        fixtures.context.valid.networkConfig,
+        fixtures.context.valid.publicKey,
+        fixtures.context.valid.privkHex
+      );
+      expect(contextOutput.privkHex).toBe(fixtures.context.valid.privkHex);
+      expect(contextOutput.privkCB58).toBe(fixtures.context.valid.privkCB58);
+    });
+
+    test('Should work when public key, private key CB58 is passed along with network config', () => {
+      const contextOutput = context(
+        fixtures.context.valid.networkConfig,
+        fixtures.context.valid.publicKey,
+        '',
+        fixtures.context.valid.privkCB58
+      );
+      expect(contextOutput.privkHex).toBe(fixtures.context.valid.privkHex);
+      expect(contextOutput.privkCB58).toBe(fixtures.context.valid.privkCB58);
+    });
+
+    test('Should throw error when invalid private key hex is passed', () => {
+      expect(() => {
+        context(
+          fixtures.context.valid.networkConfig,
+          fixtures.context.invalid.publicKey,
+          fixtures.context.invalid.privkHex
+        );
+      }).toThrowError;
+    });
+  });
+
+  describe('contextEnv Testcases', () => {
+    test('Should return context successfully', () => {
+      const context = contextEnv('.env', 'localflare');
+      expect(context).not.toBeNull;
+    });
+
+    test('Should fail for invalid network', () => {
+      expect(() => contextEnv('.env', 'invalid-network')).toThrowError('Invalid network');
+    });
+  });
+
+  describe('contextFile Testcases', () => {
+    beforeEach(() => {
+      jest.clearAllMocks();
+    });
+    test('Should return the context', () => {
+      const context = contextFile('ctx.json');
+      expect(context).not.toBeNull;
+    });
+
+    test('Should throw error as file does not exist', () => {
+      expect(() => contextFile('')).toThrowError();
+    });
+  });
+
+  describe('networkFromContextFile Testcases', () => {
+    test('Should return network for valid file', () => {
+      const network = networkFromContextFile('ctx.json');
+      expect(network).toBe('localflare');
+    });
+
+    test('Should throw error as file does not exist', () => {
+      expect(() => networkFromContextFile('')).toThrowError();
+    });
+  });
+
+  describe('getConfig Testcases', () => {
+    it('should return flare config if network is flare or undefined', () => {
```

Comment:
can you make the `it` statement consistent either start with uppercase or lower and follow the same across the test cases